agoric/versions.json

```diff
@@ -55,6 +55,44 @@
         "type": "go",
         "version": "1.20.2"
       }
+    },
+    {
+      "name": "agoric-upgrade-17",
+      "tag": "agoric-upgrade-17",
+      "proposal": 78,
+      "height": 16947291,
+      "recommended_version": "agoric-upgrade-17",
+      "compatible_versions": [
+        "agoric-upgrade-17"
+      ],
+      "cosmos_sdk_version": "agoric-labs/cosmos-sdk v0.46.16-alpha.agoric.2.4",
+      "consensus": {
+        "type": "cometbft",
+        "version": "v0.34.30",
+        "repo": "https://github.com/agoric-labs/cometbft",
+        "tag": "v0.34.30-alpha.agoric.1"
+      },
+      "cosmwasm_enabled": false,
+      "previous_version_name": "agoric-upgrade-16",
```

Comment:
because the legacy `cosmwasm_enabled` is here, we'll also require the replacement `cosmwasm` object.

Reply:
Hmm, I don't think I understand... are you asking for `"cosmwasm": { "enabled": false }`? And if so, would that be equivalent to just omitting both `cosmwasm_enabled` and `cosmwasm`?

Reply:
yes, and pretty much yes

Reply:
Done.